From c2a27b6a80c4759ecb9864d8bae6df9f5934ba4c Mon Sep 17 00:00:00 2001
From: Fabio Estevam <fabio.estevam@freescale.com>
Date: Tue, 12 Nov 2013 20:39:12 -0200
Subject: [PATCH 019/130] ARM: imx_v6_v7_defconfig: Select CONFIG_HIGHMEM

Booting a mx6q wandboard with 2GB of RAM we see the following on boot:

Booting Linux on physical CPU 0x0
Linux version 3.12.0-next-20131112+ (fabio@fabio-Latitude-E6410) (gcc version 43
CPU: ARMv7 Processor [412fc09a] revision 10 (ARMv7), cr=10c53c7d
CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
Machine model: Wandboard i.MX6 Quad Board
Truncating RAM at 10000000-8fffffff to -7f7fffff (vmalloc region overlap)
...

Select CONFIG_HIGHMEM to avoid the vmalloc region overlap.

Signed-off-by: Fabio Estevam <fabio.estevam@freescale.com>
Signed-off-by: Shawn Guo <shawn.guo@linaro.org>
---
 arch/arm/configs/imx_v6_v7_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/imx_v6_v7_defconfig b/arch/arm/configs/imx_v6_v7_defconfig
index c814e0e..f1c9bfa 100644
--- a/arch/arm/configs/imx_v6_v7_defconfig
+++ b/arch/arm/configs/imx_v6_v7_defconfig
@@ -42,6 +42,7 @@ CONFIG_VMSPLIT_2G=y
 CONFIG_PREEMPT_VOLUNTARY=y
 CONFIG_AEABI=y
 # CONFIG_OABI_COMPAT is not set
+CONFIG_HIGHMEM=y
 CONFIG_CMDLINE="noinitrd console=ttymxc0,115200"
 CONFIG_VFP=y
 CONFIG_NEON=y
-- 
1.8.5.1

